resolwe_bio/processes/workflows/mirna.yml

```diff
@@ -0,0 +1,65 @@
+- slug: workflow-mirna
+  category: workflow:primary
+  name: miRNA pipeline
+  data_name: "miRNA pipeline ({{ reads|sample_name|default('?') }})"
+  requirements:
+    expression-engine: jinja
+  version: 0.0.0
+  type: data:workflow:mirna
+  input:
+      # this set of parameters is for bowtie2
+    - name: reads
+      label: Input miRNA reads.
+      type: data:reads:fastq
+      description: >
+        Note that these reads should already be void of adapters.
+    - name: genome
+      label: Genome
+      type: data:genome:fasta
+      # this set of parameters is for featureCounts
+    - name: annotation
+      label: Annotation (GTF/GFF3)
+      type: data:annotation
+    - name: id_attribute
+      label: ID attribute
+      type: basic:string
+      default: gene_id
+      allow_custom_choice: true
+      choices:
+        - label: Gene id
+          value: gene_id
+        - label: Transcript id
+          value: transcript_id
+        - label: Gene id (GFF3)
+          value: ID
+        - label: geneid
+          value: geneid
+      description: >
+        GTF attribute to be used as feature ID. Several GTF lines with the
+        same feature ID will be considered as parts of the same feature. The
+        feature ID is used to identity the counts in the output table. If
+        you want to count by "ID" and are using GFF3 format, make sure you
+        select "Gene id (gff3)" option.
+    # this next option could perhaps be hidden and hard-coded into the workflow
+    - name: feature_type
```

Comment:
Rename to `feature_class`. See [https://github.com/genialis/resolwe-bio/pull/377/files#diff-812456a06467163f6e892c958a5dd960R58](https://github.com/genialis/resolwe-bio/pull/377/files#diff-812456a06467163f6e892c958a5dd960R58).

Reply:
Hum, this is not yet incorporated into the main branch. Should we wait and modify miRNA process until those changes are in place?

Reply:
We expect this change to be in effect within a week. I suggest the change to miRNA process now.